Ginger/Ginger/ALM/Repository/OctaneRepository.cs

```diff
@@ -55,7 +109,117 @@ public override void ExportBfActivitiesGroupsToALM(BusinessFlow businessFlow, Ob
 
         public override bool ExportBusinessFlowToALM(BusinessFlow businessFlow, bool performSaveAfterExport = false, ALMIntegration.eALMConnectType almConectStyle = ALMIntegration.eALMConnectType.Manual, string testPlanUploadPath = null, string testLabUploadPath = null)
         {
-            throw new NotImplementedException();
+            if (businessFlow == null)
+            {
+                return false;
+            }
+
+            if (businessFlow.ActivitiesGroups.Count == 0)
+            {
+                Reporter.ToUser(eUserMsgKey.StaticInfoMessage, "The " + GingerDicser.GetTermResValue(eTermResKey.BusinessFlow) + " do not include " + GingerDicser.GetTermResValue(eTermResKey.ActivitiesGroups) + " which supposed to be mapped to ALM Test Cases, please add at least one " + GingerDicser.GetTermResValue(eTermResKey.ActivitiesGroup) + " before doing export.");
+                return false;
+            }
+
+            QCTestSet matchingTS = null;
+
+            Amdocs.Ginger.Common.eUserMsgSelection userSelec = Amdocs.Ginger.Common.eUserMsgSelection.None;
+            //TO DO MaheshK : check if the businessFlow already mapped to Octane Test Suite
+            if (String.IsNullOrEmpty(businessFlow.ExternalID) == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)